Source/Laser/LaserParticleContainer.cpp

```diff
@@ -428,11 +428,11 @@ LaserParticleContainer::Evolve (int lev,
 
     amrex::LayoutData<amrex::Real>* cost = WarpX::getCosts(lev);
 
-#ifdef _OPENMP
+#ifdef AMREX_USE_OMP
 #pragma omp parallel
```

Comment:
Ok w/o `(amrex::Gpu::notInLaunchRegion())`?

Reply:
It needs `if (amrex::Gpu::notInLaunchRegion())`.

Reply:
```suggestion
#pragma omp parallel if (amrex::Gpu::notInLaunchRegion())
```